src/mlpack/methods/ann/activation_functions/gcu_function.hpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file methods/ann/activation_functions/gcu_function.hpp
+ * @author Tarek Elsayed
+ *
+ * Definition and implementation of the Growing Cosine Unit function.

```

Comment:
Can you add a reference to the paper, checkout https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/ann/activation_functions/mish_function.hpp#L8 checkoutCan you add a reference to the paper, checkout https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/ann/activation_functions/mish_function.hpp#L8 for an example.